client/state/partner-portal/licenses/test/hooks.js

```diff
@@ -320,6 +320,43 @@ describe( 'useBillingDashboardQuery', () => {
 		expect( result.current.data ).toEqual( formattedStub );
 	} );
 
+	it( 'dispatches notification on no invoice available', async () => {
```

Comment:
If we're going to stub in empty data we should have a test case or assertion that it gets returned.

Reply:
We no longer stub empty data